expression/schema.go

```diff
@@ -259,3 +260,13 @@ func MergeSchema(lSchema, rSchema *Schema) *Schema {
 func NewSchema(cols ...*Column) *Schema {
 	return &Schema{Columns: cols, TblID2Handle: make(map[int64][]*Column)}
 }
+
+// FindIndexByName finds index by name.
+func FindIndexByName(idxName string, indices []*model.IndexInfo) *model.IndexInfo {
```

Comment:
Add it directly to `model` package would be better?

Reply:
You mean move to parser/model?

Reply:
Yes, from the present usage of this function:

```
➜ ag -w FindIndexByName
util/schemautil/schemautil.go
18:// FindIndexByName finds index by name.
19:func FindIndexByName(idxName string, indices []*model.IndexInfo) *model.IndexInfo {

planner/core/planbuilder.go
756:            idx := schemautil.FindIndexByName(idxName.L, tblInfo.Indices)

ddl/index.go
189:    if fromIdx := schemautil.FindIndexByName(from.L, tbl.Indices); fromIdx == nil {
199:    if toIdx := schemautil.FindIndexByName(to.L, tbl.Indices); toIdx != nil && from.L != to.L {
211:    idx := schemautil.FindIndexByName(from.L, tblInfo.Indices)
250:    indexInfo := schemautil.FindIndexByName(indexName.L, tblInfo.Indices)
419:    indexInfo := schemautil.FindIndexByName(indexName.L, tblInfo.Indices)

ddl/db_test.go
453:            indexInfo := schemautil.FindIndexByName("idx_c2", t.Meta().Indices)

ddl/ddl_api.go
2754:   if indexInfo := schemautil.FindIndexByName(indexName.L, t.Meta().Indices); indexInfo != nil {
2895:   if indexInfo := schemautil.FindIndexByName(indexName.L, t.Meta().Indices); indexInfo == nil {

ddl/rollingback.go
71:     indexInfo := schemautil.FindIndexByName(indexName.L, tblInfo.Indices)
216:    idx := schemautil.FindIndexByName(from.L, tblInfo.Indices)
```

It would be better to bind this function to `model.TableInfo`. The function signature can be simplified to:
```go
func (t *TableInfo) FindIndexByName(idxName string) *model.IndexInfo {
	...
}
```

Reply:
@crazycs520 Please do this code refactor in another PR, let's focused on adding UT in this PR. It's better to do only one thing in a single PR.

Reply:
ok